src/app/pages/learn/index.tsx

```diff
@@ -10,25 +11,32 @@ interface LearnProps {
 }
 
 function Learn(props: LearnProps) {
-    const [ content, setContent ] = useState(null);
+    const [content, setContent] = useState(null);
 
     useEffect(
         () => {
             setContent(`Talep edilen içerik yolu: /${props.contentPath}`);
         },
-        [ props.contentPath ]
+        [props.contentPath]
     );
 
     return (
-        <>
-            <h1 className={bulmaStyles.title}>Eğitim</h1>
-
-            <p>
-                Bu sayfa yapım aşamasında.<br />
-                <br />
-                {content}
-            </p>
-        </>
+        <div>
+            <Container>
+                <Header as='h2' icon textAlign='center'>
```

Comment:
burada her ne kadar semantic ui manual'i `'` gösterse de, JSX'de attribute'lar için `"` kullanacak şekilde bir kod standartımız var.

`<Header as='h2' icon textAlign='center'>` yerine `<Header as="h2" icon textAlign="center">` diyerek özetleyebiliriz.